### DISCUSSION TITLE
many_to_one/related field name

### DISCUSSION THREAD

**Discussion by numpde** [reactive-python/reactpy-django#121] [Problem] [answered]
I have a model Compiled with a foreign key to Module, declared as follows:

```
parent = models.ForeignKey(Module, ..., related_name="compiled")
``` 

With  `use_query`, this fails because it attempts to fetch `compiled_set`, I believe, due to [this line](https://github.com/idom-team/django-idom/blob/62ea6f13743e0e9aa13f8a33b588e27b46e6462f/src/django_idom/utils.py#L235) in utils:
```
prefetch_fields.append(f"{field.name}_set")
```

Maybe it should be 
```
prefetch_fields.append(field.related_name or f"{field.name}_set")
```
?

**Comment by Archmonger** [comment, marked as answer]
Thanks. I've created a new issue for this and will fix shortly.

#122

**Reply by Archmonger** [reply]
This has been merged into main. Will be released with version `3.0.0` this weekend.

**Comment by numpde** [comment]
Thank you.
